cmd/publishing-bot/publisher.go

```diff
@@ -155,6 +155,19 @@ func (p *PublisherMunger) construct() error {
 			}
 			return strings.Join(depStrings, ",")
 		}
+		if len(repoRules.PublishScript) == 0 {
+			// TODO: Move this to main.go?
+			basePublishScriptPath := "./publish_scripts"
+			if len(os.Getenv("BASE_PUBLISH_SCRIPT_PATH")) > 0 {
+				basePublishScriptPath = os.Getenv("BASE_PUBLISH_SCRIPT_PATH")
+			}
+			repoPublishScriptPath := filepath.Join(basePublishScriptPath, "publish_"+repoRules.DestinationRepository+".sh")
+			if _, err := os.Stat(repoPublishScriptPath); err == nil {
+				repoRules.PublishScript = repoPublishScriptPath
```

Comment:
follow-up: get rid of these scripts and put the smoke tests into the rules

Reply:
https://github.com/kubernetes/publishing-bot/issues/26